x-pack/solutions/search/plugins/search_inference_endpoints/public/components/all_inference_endpoints/render_table_columns/render_endpoint/endpoint_info.tsx

```diff
@@ -8,23 +8,45 @@
 import { EuiBetaBadge, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
 import React from 'react';
 import type { InferenceInferenceEndpointInfo } from '@elastic/elasticsearch/lib/api/types';
+import { EisPromotionalTour } from '@kbn/search-api-panels';
 import { isEndpointPreconfigured } from '../../../../utils/preconfigured_endpoint_helper';
 import * as i18n from './translations';
 import { isProviderTechPreview } from '../../../../utils/reranker_helper';
+import { docLinks } from '../../../../../common/doc_links';
 
 export interface EndpointInfoProps {
   inferenceId: string;
   endpointInfo: InferenceInferenceEndpointInfo;
+  isCloudEnabled?: boolean;
 }
 
-export const EndpointInfo: React.FC<EndpointInfoProps> = ({ inferenceId, endpointInfo }) => (
+const EIS_TOUR_ENDPOINT = '.elser-2-elastic';
```

Comment:
nitpick: no need to worry about this in a Friday evening.

I would love to move this into a constants file.

i.e. `x-pack/solutions/search/plugins/search_inference_endpoints/public/components/all_inference_endpoints/constants.ts`